tests/golos.publication_rewards_tests.cpp

```diff
@@ -645,9 +630,9 @@ BOOST_FIXTURE_TEST_CASE(timepenalty_test, reward_calcs_tester) try {
     for (size_t i = 0; i < 6; i++) {    // TODO: remove magic number, 6 must be derived from some constant used in rules
         BOOST_CHECK_EQUAL(success(), addvote(_users[i], {N(bob), "permlink", ref_block_num_bob}, 10000));
         check();
-        run(seconds(3));    // TODO: remove magic number
+        produce_blocks();    // TODO: remove magic number
     }
-    run(seconds(90));       // TODO: remove magic number
+    produce_blocks(30);       // TODO: remove magic number
```

Comment:
It is a little unusual to see the number of blocks in this place. It is better to get it from seconds using the function `golos::seconds_to_blocks`